playground/tree-sitter/build.py

```diff
@@ -29,6 +29,51 @@ def run_command(cmd: List[str], cwd: Path = None) -> subprocess.CompletedProcess
         raise
 
 
+def build_concrete_syntax_wasm(repo_root: Path) -> Path:
+    """Build WASM files for concrete-syntax crate."""
+    concrete_syntax_dir = repo_root / "crates" / "concrete-syntax"
+    pkg_web_dir = concrete_syntax_dir / "pkg-web"
+    
+    print("Building concrete syntax WASM package...")
+    
+    # Check if WASM files need to be built (simple existence check)
+    wasm_files = [
+        pkg_web_dir / "concrete_syntax.js",
+        pkg_web_dir / "concrete_syntax_bg.wasm"
+    ]
+    
+    if all(f.exists() for f in wasm_files):
+        print("✓ Concrete syntax WASM files already exist, skipping build...")
```

Comment:
Is this because we want to leverage cache? But it seems to have taken out in GH actions.

We should actually check the hash of the source files + Cargo locks to ensure we are not using an old version, but that might not be worth it, perhaps we just simply always build it?